source/Applications/PointCloudTutorial.md

```diff
@@ -0,0 +1,105 @@
+
+## Introduction
+
+This tutorial describes how to use Zivid SDK to work with [Point Cloud][kb-point_cloud-url] data.
+
+1. [Frame](#frame)
+   1. [Capture](#capture)
+   2. [Load](#load)
+   3. [Visualize](#visualize)
+3.  [Point Cloud](#point-cloud)
+    1. [Get handle](#get-handle)
+    2. [Copy](#copy)
+    3.  [Transform](#transform)
+
+### Prerequisites
+
+You should have installed Zivid SDK and cloned C# samples. For more details see [Instructions][installation-instructions-url].
+
+## Frame
+
+### Capture
+
+The ```Zivid.NET.Frame``` can be captured with a camera ([go to source][frame-capture]).
+```csharp
+var frame = camera.Capture(settings);
+```
+Check [Capture Tutorial][capture-tutorial] for more details.
+
+### Load
+It can also be loaded from a ZDF file ([go to source][frame-from-file]).
+```csharp
+var frame = new Zivid.NET.Frame("/path/to/Zivid3D.zdf");
+```
+### Visualize
+
+Having the frame allows you to visualize the point cloud ([go to source][visualize-point-cloud]).
+
+```csharp
+var visualizer = new Zivid.NET.Visualization.Visualizer();
+
+visualizer.Show(frame);
+visualizer.ShowMaximized();
+visualizer.ResetToFit();
+
+visualizer.Run();
+```
+
+## Point Cloud
+
+### Get handle
+
+You can now get a handle to the point cloud data on the GPU ([go to source][point-cloud]).
+```csharp
+var pointCloud = frame.PointCloud;
+```
+The method ```Zivid.NET.Frame.PointCloud()``` does not perform any copying from GPU memory.
```

Comment:
It's not a method here, it's a "property" (note the lack of parenthesis in the snippet above).
Write something like: "Getting the property ```Zivid.NET.Frame.PointCloud``` does not perform any copying from GPU memory."